webapp/src/app/shared/widgets/character-widget/character-widget.component.scss

```diff
@@ -0,0 +1,12 @@
+@import "~@angular/material/theming";
+
+$config: mat-typography-config();
+
+.text {
```

Comment:
No need for custom styles for most widget inputs. `default-input` css class should be used